integration_tests/utils/system.go

```diff
@@ -52,7 +52,7 @@ func ExecCommand(cmd string) (string, error) {
 	substring := strings.Fields(cmd)
 	name := substring[0]
 	args := substring[1:]
-	out, err := exec.Command(name, args...).CombinedOutput()
+	out, err := exec.Command(name, args...).CombinedOutput() // #nosec G204
```

Comment:
@soniasingla Are these the only lines in the repo which give G204 warning? If not can you add comments to the other code sections also?